packages/terra-form-fieldset/src/Fieldset.scss

```diff
@@ -0,0 +1,57 @@
+:local {
+  .field,
```

Comment:
I don't think you need the field classes. I think you just need the fieldset ones.